test/perf/vector_speedtest.jl

```diff
@@ -13,28 +13,25 @@ function test(n::Int)
     @constraint(m,sum(c[i]*z[i] for i=1:n)<=0)
 
     #Vector
-    tic()
-    @constraint(m,sum(c[i]*z[i] for i=1:n)<=1)
-    println("Vector with sum(): $(toq())")
-    tic()
-    @constraint(m,vecdot(c,z) <= 1)
-    println("Vector with vecdot() : $(toq())")
+    elpsd = @elapsed @constraint(m,sum(c[i]*z[i] for i=1:n)<=1)
```

Comment:
Style nit: Don't abbreviate by deleting some vowels. `elapsed` or `run_time` is better.